docs/content/en/getting_started.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Chproxy quick start
+menuTitle: Quick start
+category: Guides
+position: 105
+---
+
+This quick start guide will walk you through an example of chproxy usage. After this guide, you will have seen how chproxy can be used to spread SELECT queries over a multi-node ClickHouse cluster.
+
+To follow along with the quick start, clone [chproxy](https://github.com/ContentSquare/chproxy) from github and navigate to the `examples/quick-start` folder.
+
+### Running the Quick Start
+
+A docker-compose file is provided to run an example ClickHouse setup, with a multi-node ClickHouse setup.
+
+Make sure your Docker Engine is provisioned with at least 4G of memory available. To start the ClickHouse servers, run:
+
+```console
+docker compose up
+```
+
+Optionally, to verify that the cluster is running properly, execute the following from a new terminal shell:
+
+```console
+docker exec -it quick_start_clickhouse_1 clickhouse-client --query "SELECT COUNT(*) FROM tutorial.views"
+```
+
+The ClickHouse server exposes an [HTTP interface](https://clickhouse.com/docs/en/interfaces/http/) on port 8123. Chproxy uses the HTTP interface to submit queries to Clickhouse.
+
+The ClickHouse servers are already setup with a Distributed Table on each node, as described in [Spreading INSERTs](/use-cases/spread-insert).
```

Comment:
I'm not sure if this relative link will work (to bad, I can't see to generated webdoc from render from your PR to test the link)

Reply:
I used this syntax as I saw more links using this relative syntax. But maybe those are the broken links mentioned in #231.